### PR TITLE
fix: move UDID between commands and params

### PR DESCRIPTION
**Pull request by dpgraham**
IDB commands seem to only work if udid comes in between the command and the parameters

_Example_
**GOOD**: `idb xctest run ui --udid <SOME_UDID> com.facebook.wda.runner com.facebook.WebDriverAgentRunner.xctrunner com.facebook.WebDriverAgentRunner.xctrunner`

**BAD**: `idb xctest run ui com.facebook.wda.runner com.facebook.WebDriverAgentRunner.xctrunner com.facebook.WebDriverAgentRunner.xctrunner --udid <SOME_UDID>`

This PR adjusts the `createSubProcess` function to address this ^

### CODE REVIEW

File: lib/tools/app-commands.js [hunk 1]
```diff
@@ -62,10 +62,10 @@ appCommands.launchApp = async function launchApp (bundleId, opts = {}) {
     return null;
   }
 
-  const processMonitor = this.createSubProcess([
-    'launch', bundleId,
-    ...args, '--wait-for',
-  ]);
+  const processMonitor = this.createSubProcess(
+    ['launch'],
```
Comment (mykola-mokhnach) [inline lib/tools/app-commands.js:66]: should command be an array? I thought this is usually a single word

**Comment by mykola-mokhnach** [comment]
Seems like e2e tests are still unhappy

**Comment by dpgraham** [comment]
@mykola-mokhnach yeah I think it's unrelated to this PR. I tried from master and got the same result: https://dev.azure.com/AppiumCI/Appium%20CI/_build/results?buildId=11265&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=59a85588-b0ba-5043-24c4-d9e29d89c6f6

I'll investigate.

**Review by KazuCocoa — APPROVED** [review]
(this change itself lgtm)

**Pull request merged**